r/NEWS.md

```diff
@@ -19,6 +19,53 @@
 
 # arrow 11.0.0.9000
 
+## New features
+
+* The `read_parquet()` and `read_arrow()` functions can now accept URL
+  arguments (#33287, #34708).
+* The `json_credentials` argument in `GcsFileSystem$create()` now accepts
+  a file path containing the appropriate authentication token (@amoeba,
+  #34421, #34524).
+* The `$options` member of `GcsFileSystem` objects can now be inspected
+  (@amoeba, #34422, #34477).
+* The `read_csv_arrow()` function now accepts literal text input wrapped in
```

Comment:
It may be worth mentioning `read_json_arrow()` too.

```suggestion
* The `read_csv_arrow()` and `read_json_arrow()` functions now accept literal text input wrapped in
```